### ISSUE TITLE
[Bug]: load data not exists file, should execute failed.

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#6325]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1399" alt="截屏2022-11-06 21 24 19" src="https://user-images.githubusercontent.com/39303074/200173294-be65b0dd-0703-45d3-b968-f419f70360c9.png">

**Comment by tianyahui-python** [comment]
<img width="1342" alt="截屏2022-11-14 10 19 24" src="https://user-images.githubusercontent.com/39303074/201561636-46d57954-d1b9-4b48-b67d-7c6324ae1252.png">

**Comment by tianyahui-python** [comment]
>Whether you should throw a nonexistent exception first? @jianwan0214 

<img width="1569" alt="截屏2022-11-14 10 18 53" src="https://user-images.githubusercontent.com/39303074/201561704-9b8b6cc8-bf62-4e37-8663-9887cf08ef76.png">

**Comment by daviszhen** [comment]
we do not focus on it today

**Comment by tianyahui-python** [comment]
to be confirmed

**Comment by domingozhang** [comment]
Try to fix in testing phase of 0.7.
If failed, reschedule in 0.8
